packages/@aws-cdk/aws-pipes-targets-alpha/package.json

```diff
@@ -89,8 +89,7 @@
     "aws-cdk-lib": "0.0.0",
     "constructs": "^10.0.0",
     "@aws-cdk/aws-pipes-alpha": "0.0.0",
-    "@aws-cdk/integ-tests-alpha": "0.0.0",
-    "@aws-cdk/aws-pipes-sources-alpha": "0.0.0"
+    "@aws-cdk/integ-tests-alpha": "0.0.0"
```

Comment:
this is the main reason for this PR, to remove `@aws-cdk/aws-pipes-sources-alpha` here.